### PR TITLE
fill in more of AbortSignal

### PR DESCRIPTION
**Pull request by autonome**

### CODE REVIEW

**Comment by jamesnw** [comment]
Is it intentional that you're removing the AbortController items? The `api.AbortSignal.abort_event` and `api.AbortSignal.aborted` items also were removed.

If you define `compat_features` in `AbortSignal.yml`, it will not combine it with the existing tagged BCD keys, so you need to add those as well.

**Comment by autonome** [comment]
> Is it intentional that you're removing the AbortController items? The `api.AbortSignal.abort_event` and `api.AbortSignal.aborted` items also were removed.
> 
> If you define `compat_features` in `AbortSignal.yml`, it will not combine it with the existing tagged BCD keys, so you need to add those as well.

Meant to mark as draft, was just pushing as a placeholder. Definitely not planning to wipe out the existing keys!

File: features/aborting.yml [hunk 1]
```diff
@@ -2,3 +2,12 @@ name: AbortController and AbortSignal
...
 caniuse: abortcontroller
+compat_features:
+  - api.AbortSignal
+  - api.AbortSignal.abort_event
+  - api.AbortSignal.abort_static
+  - api.AbortSignal.abort_static.reason_parameter
+  - api.AbortSignal.aborted
+  - api.AbortSignal.reason
+  - api.AbortSignal.throwIfAborted
+  - api.AbortSignal.timeout_static
```
Comment (ddbeck) [inline features/aborting.yml:13]: This is (still?) wiping out a bunch of keys from BCD and explicitly mentioned in the description. What's the story here?

Comment (autonome) [reply features/aborting.yml]: Ugh, I thought I'd added the difference back in but missed the AbortController section. Added back.

File: features/aborting.yml [hunk 2]
```diff
@@ -2,3 +2,16 @@ name: AbortController and AbortSignal
 description: The `AbortController` and `AbortSignal` APIs allow you to cancel an ongoing operation, such as a `fetch()` request.
 spec: https://dom.spec.whatwg.org/#aborting-ongoing-activities
 caniuse: abortcontroller
+compat_features:
```
Comment (ddbeck) [inline features/aborting.yml:5]: I looked at the additions to this and they make sense to me as later additions. However, the computed status now departs from Can I Use significantly (eventually we'll have docs for this, but see https://github.com/web-platform-dx/web-features/issues/1631#issuecomment-2310669265 for now). This needs a `compute_from` and a dist refresh:

```suggestion
status:
  compute_from: api.AbortController
compat_features:
```

**Review by ddbeck — CHANGES_REQUESTED** [review]

File: features/aborting.yml (hunk 2 above)
Comment (autonome) [reply features/aborting.yml:7]: hmmm yeah i need to add that to my checklist

**Review by ddbeck — APPROVED** [review]
Thank you!

**Pull request merged**